javalib/src/main/scala/java/util/concurrent/ThreadLocalRandom.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Written by Doug Lea with assistance from members of JCP JSR-166
+ * Expert Group and released to the public domain, as explained at
+ * http://creativecommons.org/publicdomain/zero/1.0/
+ *
+ * and translated to Scala
+ */
+
+package java.util.concurrent
+
+import java.util.Random
+
+class ThreadLocalRandom private (var initialized: Boolean = false)
```

Comment:
Don't put `initialized` there. Put it as a `private var` in the body. However, do *not* assign it, i.e.:
```scala
private var initialized: Boolean = _
```
If you assign it (to `false`), it will be reassigned to `false` after the super constructor has been called. But it is that super constructor which calls `setSeed` which is supposed to set it to true, so the reset to `false` destroys the work.

Alternatively, remove `initialized = true` from `setSeed` and set it to `true` in the constructor (this is cleaner, IMO):
```scala
class ThreadLocalRandom ... {
  private var initialized: Boolean = _
  initialized = true // executed after the super constructor is done
```